enrollment-server-onboarding-api-model/src/main/java/com/wultra/app/enrollmentserver/api/model/onboarding/response/data/ConfigurationDataDto.java

```diff
@@ -31,4 +31,9 @@ public class ConfigurationDataDto {
      */
     private String otpResendPeriod;
```

Comment:
We may mark this property as deprecated, create an issue for removal and link it from here something like `// TODO (dusil, 2023-08-10, #xxx) some explanation`